PayPalNativeCheckout/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation deps.appCompat
     implementation project(':PayPalDataCollector')
 
-    implementation('com.paypal.checkout:android-sdk:0.8.8') {
+    implementation('com.paypal.checkout:android-sdk:1.0.0') {
```

Comment:
q: is this version GA yet? I'm unable to find it on [mvnrepository.com](https://mvnrepository.com/artifact/com.paypal.checkout/android-sdk).

Reply:
nah it was just pushed late last night haha it should be live now though